src/components/Pagination/readme.md

```diff
@@ -58,10 +79,10 @@ Pagination base:
                 <div>
                     <GlobalHeader src="images/user/user3.jpg" />
                     <div className="rainbow-p-around_x-large rainbow-p-around_x-large">
-                        <div>
+                        <div className="rainbow-flex">
                             {this.getContent()}
                         </div>
-                        <Pagination className="rainbow-m_auto" pages={6} activePage={activePage} onChange={this.handleOnChange} />
+                        <Pagination className="rainbow-m_auto" pages={4} activePage={activePage} onChange={this.handleOnChange} />
                     </div>
```

Comment:
the arrows in the pagination componet is not centered
<img width="334" alt="screen shot 2018-10-27 at 6 41 52 pm" src="https://user-images.githubusercontent.com/32083556/47610327-259e4600-da18-11e8-9d82-1b9d899b0f9a.png">